content/blog/2016/2016-03-11-new-jenkins-20-preview.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "New Jenkins 2.0 Preview Build"
```

Comment:
This could maybe be a bit more specific (i.e. mention alpha 3), otherwise every new preview post will have this same title :)